File: yh_turtle/src/yh_turtle_move.cpp
#include "ros/ros.h"
#include "geometry_msgs/Twist.h"

// 메인 함수
int main(int argc, char ** argv)
{
    ros::init(argc, argv, "yh_turtle_move");
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<geometry_msgs::Twist>("/turtle1/cmd_vel", 100);
    ros::Rate loop_rate(1);

    geometry_msgs::Twist msg;
    msg.angular.z = -0.2;
    msg.linear.x = 1;

    while (ros::ok())
    {
        pub.publish(msg); // pub이 msg를 퍼블리시한다.
        loop_rate.sleep();
    }
    
    return 0;
}

File: yh_turtle/scripts/yh_turtle_move.py
#!/usr/bin/python
#-*- coding: utf-8 -*-

import turtle
import rospy
from geometry_msgs.msg import Twist
    #패키지의 메세지에서 우리가 원하는 Twist 꺼내오기

def talker():
    rospy.init_node("yh_turtle_move")
    pub = rospy.Publisher("/turtle1/cmd_vel", Twist, queue_size = 100)
    msg = Twist()
    msg.linear.x = 4
    loop_rate = rospy.Rate(1)

    while not rospy.is_shutdown():
        pub.publish(msg)
        loop_rate.sleep()

if __name__ == "__main__":
    try:
        talker()
    except rospy.ROSInterruptException:
        pass